<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="changeWindowButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>361</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>CARGAR SISTEMA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Raspberry</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rasp_connected">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;No conectado&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>371</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>371</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>376</width>
        <height>361</height>
       </rect>
      </property>
      <property name="text">
       <string>CARGANDO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_rasp_connected_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>370</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;No conectado&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadSystemButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>361</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>CARGAR SISTEMA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeLightPositionTest">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>150</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Resumen</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_luz_izq">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonLeftLight">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Luz izq.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_luz_drc1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="changeButtonLights">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>200</width>
       <height>32</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Cambiar posicion luz/luces</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>370</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Raspberry</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRightLight">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>luz drc.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>LUZ DRC</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>LUZ IZQ</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonTwoLights">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>150</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Ambas/juntar luces</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
